G1ANT.Addon.OCR.Tesseract/G1ANT.Addon.OCR.Tesseract/Commands/OcrOfflineFindCommand.cs

```diff
@@ -37,31 +38,48 @@ public class Arguments : CommandArguments
 
             [Argument(Tooltip = "The language which should be considered trying to recognize text")]
             public TextStructure Language { get; set; } = new TextStructure("eng");
+
+            [Argument(Tooltip = "The ratio used for rescaling of the image before doing actual OCR. Default is 2.0. Higher values are better for recognizing details like small fonts.")]
+            public FloatStructure Sensitivity { get; set; } = new FloatStructure(2.0);
         }
         public OcrOfflineFindCommand(AbstractScripter scripter) : base(scripter)
         {
         }
 
-        double imgRescaleRatio = 4.0;
         public void Execute(Arguments arguments)
         {
-            var rectangle = !arguments.Relative.Value ? arguments.Area.Value : arguments.Area.Value.ToAbsoluteCoordinates();
+            if (!arguments.Area.Value.IsValidRectangle())
+                throw new ArgumentException("Argument Area is not a valid rectangle");
+            var rectangle = arguments.Area.Value;
+            if (arguments.Relative.Value)
+            {
+                var foregroundWindowRect = new RobotWin32.Rect();
+                RobotWin32.GetWindowRectangle(RobotWin32.GetForegroundWindow(), ref foregroundWindowRect);
+                rectangle = new Rectangle(rectangle.X + foregroundWindowRect.Left,
+                    rectangle.Y + foregroundWindowRect.Top,
+                    foregroundWindowRect.Right - foregroundWindowRect.Left,
+                    foregroundWindowRect.Bottom - foregroundWindowRect.Top);
```

Comment:
Why we have these 2 lines here? Width and Height should be from rectangle provided by user

Reply:
done